Apps/Playground/Scripts/config.json

```diff
@@ -183,7 +183,7 @@
         },
         {
             "title": "LightFalloff",
-            "playgroundId": "#20OAV9#201",
+            "playgroundId": "#20OAV9#11218",
             "referenceImage": "lightFalloff.png"
         },
```

Comment:
This doesn't seem to match what's in Babylon.js anymore. Should we update it?

Reply:
A new pass on tests should be done. Some are updated in bjs and not here. some are replaced,...